src/ovh-password/index.js

```diff
@@ -28,6 +29,12 @@ angular
       animation: 'flat-fade',
     });
   })
-  .run(/* @ngTranslationsInject ./translations */);
+  .run(/* @ngTranslationsInject ./translations */)
+  .run(/* @ngInject */ ($templateCache) => {
```

Comment:
Not needed for this case, is it ?

https://github.com/schmod/babel-plugin-angularjs-annotate#implicit-arrow-function-annotation

Reply:
You're right!

**With** the `/* @ngInject */`:
```js
}]).run(
/* @ngInject */
["$templateCache", function ($templateCache) {
  $templateCache.put('telecomUniverseComponents/ovh-password/strength/check/ovh-password-strength-check.html', tucOvhPasswordStrengthCheckTemplate);
}]);
```

**Without**:
```js
}]).run(["$templateCache", function ($templateCache) {
  $templateCache.put('telecomUniverseComponents/ovh-password/strength/check/ovh-password-strength-check.html', tucOvhPasswordStrengthCheckTemplate);
}]);
```

I prefer the explicit version.

Reply:
We had a talk with @cbourgois and we agreed to be explicit every time